pkgs/development/python-modules/fbtftp/default.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, buildPythonPackage, fetchPypi }:
+buildPythonPackage rec {
+  pname = "fbtftp";
+  version = "0.4";
+
+  doCheck = false;
```

Comment:
Why did you disable tests?

Reply:
@SuperSandro2000 Hi and thank you for this question, because it was not working in my nix-shell without disable this parameter. Certainly due to some network behavior that seems not related to this package.

Reply:
It's fine to disable tests, please just add a comment as to why they were disabled. Networking or other behavior which can't be done in a sandbox is reasonable.

Reply:
Hi, thank you for this answer, understood, I will add the comment then.

Reply:
@French-isotope: I had a second look at the test, and actually, the network issue was because the installer tried to fetch libraries for build using pip.

You can remove the `doCheck = false` and instead provides the required dependencies:

```
           checkInputs = [ mock overage nose busybox ];
```

`busybox` is not mandatory, if not present, one test is skiped.

Reply:
> @French-isotope: I had a second look at the test, and actually, the network issue was because the installer tried to fetch libraries for build using pip.
> 
> You can remove the `doCheck = false` and instead provides the required dependencies:
> 
> ```
>            checkInputs = [ mock overage nose busybox ];
> ```
> 
> `busybox` is not mandatory, if not present, one test is skiped.

@guibou Nice ! How did you know where to look to know that ?